google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingHandler.java

```diff
@@ -310,12 +354,15 @@ private LogEntry entryFor(LogRecord record) {
         .addLabel("levelName", level.getName())
         .addLabel("levelValue", String.valueOf(level.intValue()))
         .setSeverity(severityFor(level));
+
     enhanceLogEntry(builder, record);
     return builder.build();
   }
   
   protected void enhanceLogEntry(LogEntry.Builder builder, LogRecord record) {
```

Comment:
I kept it there because it is a released API.  But it only existed for a very short period of time...
How about I deprecate it and move the behaviour out of it.